fcm.go

```diff
@@ -40,6 +43,12 @@ var (
 	fcmServerUrl = fcm_server_url
 )
 
+var firebaseApp *firebase.App
```

Comment:
Just curious when you run it locally I suppose this would be assigned a value somewhere?

Reply:
yep exactly